GNUmakefile

```diff
@@ -123,6 +123,22 @@ pkg/windows_amd64/nomad: $(SOURCE_FILES) ## Build Nomad for windows/amd64
 		-tags "$(GO_TAGS)" \
 		-o "$@.exe"
 
+pkg/linux_ppc64le/nomad: $(SOURCE_FILES) ## Build Nomad for linux/arm64
```

Comment:
Let's make it clear these are not part of our normal release process (and fix a typo):
```suggestion
# Not built as part of normal release process
pkg/linux_ppc64le/nomad: $(SOURCE_FILES) ## Build Nomad for linux/ppc64le
```

Reply:
Thanks that's fine.